package.json

```diff
@@ -26,7 +26,8 @@
     "minimist": "^1.1.1",
     "nightwatch": "^1.0.6",
     "plugin-error": "^1.0.1",
-    "shell-quote": "^1.4.3"
+    "shell-quote": "^1.4.3",
+    "through2": "^3.0.0"
```

Comment:
`through2` -> `through` ?

In addition, I think that `event-stream` package is no longer needed.

Reply:
True! I'll did that a bit in a hurry, sorry for that.
I'll update the `package.json`